plugins/flytekit-flyin/flytekitplugins/flyin/vscode_lib/constants.py

```diff
@@ -23,3 +23,5 @@
 # The path is hardcoded by code-server
 # https://coder.com/docs/code-server/latest/FAQ#what-is-the-heartbeat-file
 HEARTBEAT_PATH = os.path.expanduser("~/.local/share/code-server/heartbeat")
+
+INTERACTIVE_DEBUGGING_FILE_NAME = "interactive_debug.py"
```

Comment:
maybe we should make the name a bit special so users know it is generated. how about `_generated_interactive_debug.py`